android/src/main/java/com/expensify/livemarkdown/MarkdownStyle.java

```diff
@@ -95,10 +103,13 @@ public MarkdownStyle(@NonNull ReadableMap map, @NonNull Context context) {
     mPreBackgroundColor = parseColor(map, "pre", "backgroundColor", context);
     mMentionHereColor = parseColor(map, "mentionHere", "color", context);
     mMentionHereBackgroundColor = parseColor(map, "mentionHere", "backgroundColor", context);
+    mMentionHereBorderRadius = parseFloat(map, "mentionHere", "borderRadius") * screenDensity;
```

Comment:
I'd rather keep the original value of `borderRadius` in `MarkdownStyle` and multiply it by `screenDensity` near the drawing logic.